week1/readme.md

```diff
@@ -31,6 +32,18 @@
       - `node -v`
       - `npm -v`
       - In either case you should see the version of `node` or `npm` installed.
+      
+## Tests
```

Comment:
Try to introduce tests. I don't know where it is best to try to do this introduction--let me know if there might be a better way of doing this.

Reply:
I think this is a good place for the tests! The description is very basic, but i think good enough to get started. We could also add a video that just shortly explains the concept